group_vars/dev.yml

```diff
@@ -261,6 +261,15 @@ users:
     shell: /bin/bash
     update_password: on_create
 
+  - username: jmahy
+    name: "James Mahy"
+    groups:
+      - docker
```

Comment:
some instances will not have the docker group, add this to groups to create so that it's always there